src/main/java/frc/robot/Robot.java

```diff
@@ -406,6 +401,25 @@ public void teleopRoutine() {
 
       this.drive.drive(forward, turn); 
 
+      double intake, outake;
+      
+      intake = this.operatorPad.getValue(ControllerBindings.INTAKE, ControllerBindings.LEFT_TRIGGER_AXIS);
+      outake = this.operatorPad.getValue(ControllerBindings.OUTTAKE, ControllerBindings.RIGHT_TRIGGER_AXIS);
+
+      // Intake
+      if (intake != 0){
```

Comment:
Comparing to 0 here means the trigger will be very, very touchy and prone to accidental presses or even just physical binding of the trigger mechanism causing the reading to stick above 0. Probably better to compare to a deadband of 0.05 or 0.10 here to be more sure the trigger is being intentionally pulled, same as we normally do for joysticks. Or compare to a larger value like `0.9` if you want the action to only occur when the trigger is "fully" pulled.

Reply:
right, meant to change that, will be done right now